### ISSUE TITLE
Files app contains duplicate IDs

### ISSUE THREAD

**Issue by Henni** [owncloud/core#18126]
The files app uses the same ids multiple times.

For example the id `fileList` can be found 5 times.
This happens because there exists a list for every navigation point in the sidebar ('All files', 'Favorites', ...).

This could be solved by replacing all duplicate ids with classes.

**Comment by karlitschek** [comment]
@PVince81

**Comment by PVince81** [comment]
Yes it could... would probably need a major refactoring because there a still obscure places that might need it.

Ideally I'd like to get rid of IDs completely and only rely on classes, except for the main container.

**Comment by LukasReschke** [comment]
Also https://github.com/owncloud/core/issues/14536
